includes/ApplHandlingByStr.php

```diff
@@ -15,11 +15,11 @@ function showStatus( $stat_str )
 	for ( $i =  1;  $i < count ( $STR ) ; $i++ )
 	{
 		$str = explode ( ',', $STR[$i] );
-		$usr = $str[0];
+		
```

Comment:
__[code_sniffer]__
* Spaces must be used to indent lines; tabs are not allowed

<!-- Comment by [SideCI](https://sideci.com) -->